build/tf-wrapper.sh

```diff
@@ -0,0 +1,101 @@
+##TODO:  better error handling
+##TODO:  variable for path?
+##TODO:  How does this work for monorepo?
+##TODO:  
+
+#!/bin/bash
+branchname=$2
+policyrepo=$3
+
+tf_validator_path="./terraform-validator-linux-amd64"
```

Comment:
@rjerrems I see that we have this dockerfile https://github.com/terraform-google-modules/terraform-google-bootstrap/blob/master/modules/cloudbuild/cloudbuild_builder/Dockerfile that is built, can we add tf-validator bin in there?

Reply:
Yeah agree we should include that as a part of that container that gets built

Reply:
qq: Should this wrapper call this script that finds the credentials?: https://github.com/terraform-google-modules/terraform-google-bootstrap/blob/master/modules/cloudbuild/cloudbuild_builder/entrypoint.bash

There is a point at which the cicd tool (either Jenkins or Cloud Build) has to impersonate the terraform Service account. I am not sure exactly when this is happening in this script